resolver/blocking_resolver_test.go

```diff
@@ -1130,78 +1127,4 @@ var _ = Describe("BlockingResolver", Label("blockingResolver"), func() {
 			})
 		})
 	})
-
-	Describe("Redis is configured", func() {
```

Comment:
Would be nice to keep this, and other tests that validate redis use. You know better if it's feasible, but just wanted to create a thread and see if there's something we can do about it.

Reply:
Not possible without a complete rewrite of the test itself since it is based on miniredis which is not compatible to rueidis.

The logic regarding the redis usage will change significantly so it would be more reasonable to develop new unit tests for the new logic.

Reply:
Ok, you know better than me :)